src/element-stub.js

```diff
@@ -15,14 +15,24 @@
  */
 
 import {BaseElement} from './base-element';
-
+import {getMode} from '../src/mode';
 
 /** @type {!Array} */
 export const stubbedElements = [];
 
+let ampAdScriptInserted = false;
+
+export function resetAdScriptInserted() {
+  ampAdScriptInserted = false;
+}
+
 export class ElementStub extends BaseElement {
   constructor(element) {
     super(element);
+    if (toAddAdScript(element, this.getWin())) {
+      const ampAdScript = getAdScript(this.getWin());
+      this.getWin().document.head.appendChild(ampAdScript);
```

Comment:
I set it in function toAddAdScript(). want to set ampAdScriptInserted to true in the case amp-ad script is already in head. Do I need to move it outside the function?